gamestonk_terminal/forex/fx_view.py

```diff
@@ -0,0 +1,544 @@
+        df = pd.DataFrame.from_dict(response["trades"])
+        df = df[
+            [
+                "id",
+                "instrument",
+                "initialUnits",
+                "currentUnits",
+                "price",
+                "unrealizedPL",
+            ]
+        ]
+        df = df.rename(
+            columns={
+                "id": "ID",
+                "instrument": "Instrument",
+                "initialUnits": "Initial Units",
+                "currentUnits": "Current Units",
+                "price": "Entry Price",
+            }
+        )
+        print(df)
+    except V20Error as e:
+        print(e)
+
+
+def close_trade(accountID, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="close_trade",
+        description="close a trade",
+    )
+    parser.add_argument(
+        "-i",
+        "--id",
+        dest="orderID",
+        action="store",
+        type=str,
+        required=True,
+    )
+    parser.add_argument(
+        "-u",
+        "--units",
+        dest="units",
+        action="store",
+        type=str,
+        required=False,
+    )
+    ns_parser = parse_known_args_and_warn(parser, other_args)
+    if not ns_parser:
+        return
+    data = {
+        "units": ns_parser.units,
+    }
+    try:
+        request = trades.TradeClose(accountID, ns_parser.orderID, data)
+        response = client.request(request)
+        print(response)
+    except V20Error as e:
+        print(e)
+
+
+def show_candles(accountID, instrument, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="show_candles",
+        description="Display Candle Data",
+    )
+    parser.add_argument(
+        "-g",
+        "--granularity",
+        dest="granularity",
+        action="store",
+        type=str,
+        default="D",
+        required=False,
+    )
+    parser.add_argument(
+        "-c",
+        "--count",
+        dest="candlecount",
+        action="store",
+        default=180,
+        type=int,
+        required=False,
+    )
+
+    ns_parser = parse_known_args_and_warn(parser, other_args)
+    if not ns_parser:
+        return
+
+    parameters = {}
+    parameters["granularity"] = ns_parser.granularity
+    parameters["count"] = ns_parser.candlecount
+    try:
+        request = instruments.InstrumentsCandles(instrument, params=parameters)
+        response = client.request(request)
+        process_candle_response(response)
+        oanda_fix_date(".temp_candles.csv")
+        df = pd.read_csv(".candles.csv", index_col=0)
+        df.index = pd.to_datetime(df.index)
+        df.columns = ["Open", "High", "Low", "Close", "Volume"]
+        if gtff.USE_ION:
+            plt.ion()
+        mpf.plot(
+            df,
+            type="candle",
+            style="charles",
+            volume=True,
+            title=f"{instrument} {ns_parser.granularity}",
+        )
```

Comment:
It should fail more gracefully than:
```
(✨) (fx)> candles
{"errorMessage":"Invalid value specified for 'instrument'"}
```

by printing the error message and a new line.